.circleci/scripts/test_build.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+
+if [[ "$OSTYPE" == "msys" ]]; then
+    # Install python version and create a virtuallenv
+    nuget install python -Version $PYTHON_VERSION -ExcludeVersion -OutputDirectory .
+    ./python/tools/python.exe --version
+    ./python/tools/python.exe -m pip install virtualenv
+    ./python/tools/python.exe -m virtualenv env
+    # When running script under Windows executor we need to activat the venv
+    # created for the specific Python version
+    source env/Scripts/activate
+fi
+
+# Install required dependencies
+# DEV: `pyopenssl` needed until the following PR is released
+#      https://github.com/pypa/twine/pull/447
+# DEV: `wheel` is needed to run `bdist_wheel`
+pip install twine readme_renderer[md] pyopenssl wheel cython
+# Ensure we didn't cache from previous runs
+rm -rf build/ dist/
+# Manually build any extensions to ensure they succeed
+python setup.py build_ext --force
+# Ensure source package will build
+python setup.py sdist
+# Ensure wheel will build
+python setup.py bdist_wheel
+# Ensure package long description is valid and will render
+# https://github.com/pypa/twine/tree/6c4d5ecf2596c72b89b969ccc37b82c160645df8#twine-check
+twine check dist/*
```

Comment:
I feel like that this script should be reused for all platforms. Can we share this and make sure they all have the same process?

Reply:
It's being used by all platforms now https://github.com/DataDog/dd-trace-py/pull/1540/files#diff-1d37e48f9ceff6d8030570cd36286a61R145